packages/slice-machine/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -206,3 +216,27 @@ export const WithBoxContent = {
   },
   parameters: { controls: { hideNoControlsWarning: true, include: [] } },
 };
+
+type OrganizationSelectProps = { disabled?: boolean };
+
+function OrganizationSelect(props: OrganizationSelectProps): ReactNode {
```

Comment:
"Organization" is a GitHub-specific term and is only one kind of owner type (the other is "User"). The API uses the term "Owner" to represent an organization or user. Thus, we could replace references to "Organization" with "Owner". What do you think?